Unix/configure

```diff
@@ -2298,6 +2300,12 @@ then
     cp $rootrelative/etc/omigen.conf \$destdir/$sysconfdir/omigen.conf
 fi
 
+
+if test ! -f "\$destdir/$sysconfdir/omireg.conf"
+then 
+    cp $rootrelative/etc/omireg.conf \$destdir/$sysconfdir/omireg.conf
+fi
+
```

Comment:
I'm not fully clear on why this is conditional, since at line 6383 you seem to be copying unconditionally. Can you explain why this is conditional please?